data-prepper-plugins/geoip-processor/build.gradle

```diff
@@ -25,6 +31,73 @@ dependencies {
     testImplementation project(':data-prepper-test-common')
 }
 
+task copyGeoLite2Files(type: Copy) {
+    def urls = [
+            'https://github.com/maxmind/MaxMind-DB/tree/main/test-data/GeoLite2-City-Test.mmdb',
+            'https://github.com/maxmind/MaxMind-DB/tree/main/test-data/GeoLite2-Country-Test.mmdb',
+            'https://github.com/maxmind/MaxMind-DB/tree/main/test-data/GeoLite2-ASN-Test.mmdb'
+    ]
+
+    def geoLite2Dir = file('/src/test/resources/mmdb-file/geo-lite2')
+
+    from(urls.collect { url ->
+        def testFileName = url.substring(url.lastIndexOf('/') + 1)
+        def testMmdbSubString = testFileName.substring(testFileName.lastIndexOf('-'))
+        def fileName = testFileName.substring(0, testFileName.length() - testMmdbSubString.length())
+        ant.get(src:url, dest: "$geoLite2Dir/$fileName" + ".mmdb")
+    })
+
+    into(geoLite2Dir)
+
+    doFirst {
+        // Trust all certificates
```

Comment:
Why do you need to trust all? The GitHub certificate is valid.

For example, a simple download works just fine:

```
wget https://github.com/maxmind/MaxMind-DB/tree/main/test-data/GeoLite2-City-Test.mmdb
```